### PR TITLE
website: update readme

### PR DESCRIPTION
**Pull request by zchsh**
This PR updates the `website/README.md` with the latest shared documentation blocks, using the `npm run generate:readme` command.

---

Previously these blocks had been manually updated for `packer` specifically - now, with `docs-sidenav` changes rolled out to all project sites, we can used shared documentation blocks from `nextjs-scripts`.

This PR also bumps `nextjs-scripts` to the [`18.2.0` release of `nextjs-scripts`](https://github.com/hashicorp/nextjs-scripts/releases/tag/18.2.0).

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -149,7 +153,7 @@ $ curl ...
 </Tab>
 </Tabs>
 
-Continued normal markdown content
+Contined normal markdown content
```
Comment (jescalan) [inline website/README.md:156]: Typo here

**Review by jescalan — APPROVED** [review]

File: website/README.md (hunk 1 above)
Comment (zchsh) [reply website/README.md:156]: Ah dang - looks like we'll need to patch in `nextjs-scripts`, right?

Comment (jescalan) [reply website/README.md:156]: correct - we can flow this into the next release if thats easier and not let it hold up this PR

**Review by nywilken — APPROVED** [review]
LGTM

File: website/README.md (hunk 1 above)
Comment (zchsh) [reply website/README.md:156]: Sweet - copied from another PR, I've made an [Asana task](https://app.asana.com/0/1100423001970639/1200233780102792/f) and have opened [a PR on nextjs-scripts](https://github.com/hashicorp/nextjs-scripts/pull/161) with typo fixes.

**Pull request merged**